adobe-dmeservices/user-sync.py
# DMESVCS-576 # Add test case for read_desired_user_groups() in rules.py

## Summary
* Added a new test case for read_desired_user_groups


## Testing Steps
* Execute test case test_read_desired_user_groups() in test_rules.py file which should run successfully without any error



Fixes #199
